strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/AbstractArtifactManagementService.java

```diff
@@ -132,7 +132,6 @@ private long storeArtifact(RepositoryPath repositoryPath,
         String storageId = storage.getId();
 
         String artifactPathRelative = repositoryPath.getResourceLocation();
```

Comment:
All across the code base this is actually referred to as just `path`. It's always a relative path. I think it would be best to have it as just `path` here as well.